website/docs/cloud-docs/integrations/aws-service-catalog/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+page_title: Terraform Cloud for AWS Service Catalog
+description: >-
+  Provision infrastructure with HashiCorp Terraform Cloud inside AWS Service Catalog
```

Comment:
Quick question / suggestion: Should we generalize this a bit since it's technically doable in both TFC and TFE?
```suggestion
  Provision infrastructure with HashiCorp Terraform Cloud or Terraform Enterprise inside AWS Service Catalog
```

Reply:
https://github.com/hashicorp/terraform-docs-common/pull/507#discussion_r1414335429